docs/integrators/deep-history-squad.md

```diff
@@ -149,41 +156,30 @@ gcloud storage cp gs://multiversx-deep-history-archives-mainnet/shard-0/Epoch_0{
 
 ## Reconstructing non-pruned databases
 
-An alternative to downloading the non-pruned history is to reconstruct it locally (on your own infrastructure). The reconstruction process is, generally speaking, based on the **[import-db](/validators/import-db/)** feature, which re-processes past blocks - and, while doing so, retains the whole, non-pruned accounts history. However, it's also possible to reconstruct the history by performing a regular sync (e.g. from Genesis) using a properly configured deep-history observer.
+An alternative to downloading the non-pruned history is to reconstruct it locally (on your own infrastructure). 
 
-First, you need to decide whether to reconstruct a **complete** or a **partial** history. A _complete_ history provides the deep-history squad the ability to resolve historical account (state) queries **up until the Genesis**. A _partial_ history, instead, allows it to resolve state queries up until **a chosen past epoch** or **between two chosen epochs**.
+There are also two options for reconstructiong a non-pruned database:
+- Based on the **[import-db](/validators/import-db/)** feature, which re-processes past blocks - and, while doing so, retains the whole, non-pruned accounts history.
+- By performing a regular sync from the network (e.g. from Genesis), using a properly configured deep-history observer.
 
 :::note
 The reconstruction flow has to be performed **for each shard, separately**.
 :::
 
+### Reconstructing using import-db
+
+First, you need to decide whether to reconstruct a **complete** or a **partial** history. A _complete_ history provides the deep-history squad the ability to resolve historical account (state) queries **up until the Genesis**. A _partial_ history, instead, allows it to resolve state queries up until **a chosen past epoch** or **between two chosen epochs**.
+
 [comment]: # (mx-context-auto)
 
-### Reconstruct a complete history
+#### Reconstruct a complete history
 
 :::note
-Below, the reconstruction is exemplified for **mainnet, shard 0**. The same applies to other networks (devnet, testnet) and shards (including the metachain). Furthermore, the example refers to the latest Protocol release as of January 2024 (Sirius). When reconstructing the history, use the latest release available.
+Below, the reconstruction is exemplified for **mainnet, shard 0**. The same applies to other networks (devnet, testnet) and shards (including the metachain). When reconstructing the history, use the latest release available.
```

Comment:
> When reconstructing the history, use the latest release available.

Now, it's a bit redundant. But can also stay.

Reply:
Done.